src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+#spring:
+#  datasource:
+#    username: root
+#    password: secret
+#    url: jdbc:mysql://localhost:3306/phonebook
+#    driver-class-name: com.mysql.cj.jdbc.Driver
+#  jpa:
+#    show-sql: true
+#    hibernate:
+#      ddl-auto: update
+#    properties.hibernate.dialect: org.hibernate.dialect.MySQL5Dialect
```

Comment:
Ok, what is missing:
- unit tests. Show the power of IoC/DI in unit testing!
- a README file which shall describe the application, how it works, what is the way to launch it and how-to use it at all.

Reply:
I added some unit tests in `PhonebookmvcApplicationTests` class. And I added README file with some description.